flink-core/src/main/java/org/apache/flink/configuration/RestartStrategyOptions.java

```diff
@@ -89,11 +89,10 @@ public class RestartStrategyOptions {
                                                             "here")))
                                     .text(
                                             "If checkpointing is disabled, the default value is %s. "
-                                                    + "If checkpointing is enabled, the default value is %s with %s restart attempts and '%s' delay.",
+                                                    + "If checkpointing is enabled, the default value is %s, and the values of all %s related options have not changed.",
```

Comment:
Hi @zhuzhurk , do you have any good suggestion about the doc of default restart strategy?

Reply:
maybe "and the values of all %s related options have not changed." -> "and the default values of %s related config options will be used."

Reply:
Thanks for the suggestion! It's better than before, I updated.